src/main/java/com/purbon/kafka/topology/quotas/QuotasManager.java

```diff
@@ -0,0 +1,58 @@
+package com.purbon.kafka.topology.quotas;
+
+import com.purbon.kafka.topology.Configuration;
+import com.purbon.kafka.topology.model.User;
+import com.purbon.kafka.topology.model.users.Quota;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.common.quota.ClientQuotaAlteration;
+import org.apache.kafka.common.quota.ClientQuotaEntity;
+import org.apache.kafka.common.quota.ClientQuotaFilter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class QuotasManager {
+
+  private static final Logger LOGGER = LogManager.getLogger(QuotasManager.class);
+
+  private final AdminClient adminClient;
+  // private final TopologyBuilderAdminClient adminClient;
+  private final Configuration config;
+
+  public QuotasManager(AdminClient adminClient, Configuration config) {
```

Comment:
Should this be getting instead of AdminClient, the internal wrapper **TopologyAdminClient** ? This is how other classes work by wrapping there all internals of AdminClient usage. What do you think?

Reply:
I think that it's the better option, encapsulate the operations inside the TopologyAdminClient. 
In the first moment, I think to avoid dirtying the TopologyAdminClient with maybe occasional operations. Maybe adding an Interface to avoid propagate methods to other parts of the code where are not necessary.
I change it!!!!